core/src/main/scala/org/bitcoins/core/wallet/builder/RawTxFinalizer.scala

```diff
@@ -361,33 +361,97 @@ object ShufflingNonInteractiveFinalizer
   }
 }
 
-case class SubtractFeeFromOutputsFinalizer(
-    inputInfos: Vector[InputInfo],
-    feeRate: FeeUnit)
+/** Adds a future fee amount to the output with the given
+  * ScriptPubKey and subtracts that amount in equal portions
+  * from the specified change ScriptPubKeys.
```

Comment:
nit (or maybe a misunderstanding on my part): Add to scaladoc 

>This is complentary to `SubtractFromOutputFinalizer` in the sense that an [[futureFee]] is an _absolute_ fee rather than a fee rate.